### PR TITLE
Documentation draft

### PR DESCRIPTION
**Pull request by mjste**

### CODE REVIEW

**Review by my-alaska — APPROVED** [review]

File: doc/Makefile [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Minimal makefile for Sphinx documentation
+#
+
```
Comment (j-adamczyk) [inline doc/Makefile:3]: Directory name: shouldn't this be `docs`, not `doc`?

File: doc/conf.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = "scikit-fingerprints"
+copyright = "2024, scikit-fingerprints team"
+author = "scikit-fingerprints team"
+release = "0.1.2"
```
Comment (j-adamczyk) [inline doc/conf.py:12]: This should be 1.2.0

**Review by j-adamczyk — APPROVED** [review]

File: doc/Makefile (hunk 1 above)
Comment (mjste) [reply doc/Makefile]: changed now

File: doc/conf.py (hunk 2 above)
Comment (mjste) [reply doc/conf.py]: Done

**Pull request merged**